straxen/url_config/dispatcher.py

```diff
@@ -0,0 +1,85 @@
+from .regex_dispatcher import RegexDispatcher
+
+import re
+import warnings
+
+check_urls = RegexDispatcher("check_urls")
+
+
+@check_urls.register(r".*")
+def confirm_xedocs_schema_exists_check(url: str):
+    import xedocs
+    # Selects the schema name in the URL and confirms it exists in the xedocs db
+    if not re.findall(r"(?<=\://)[^\://]*(?=\?)", url)[0] in xedocs.list_schemas():
+        warnings.warn("You tried to use a schema that is not part of the xedocs database")
+
+
+@check_urls.register(r".*(list-to-array|'object-to-dict').*")
+def data_as_list_check(url: str):
+    if not ("as_list=True" in url):
+        warnings.warn("When using the list-to-array or object-to-dict protocol, you must include an as_list=True in the URL arguments")
+    if not ("sort" in url):
+        warnings.warn("When using the list-to-array or object-to-dict protocol, you must include a sort argument in the URL")
+
+
+@check_urls.register(r".*")
+def are_resources_needed_check(url: str):
+    parent_class = get_parent_class(url)
+
+    if ("BaseMap" == parent_class) or ("BaseResourceReference" == parent_class):
+        if not ("resource://" in url):
+            warnings.warn("A URL which requeres the resource:// was given however resource:// was not found within the URL, not data will be loaded.")
+
+
+@check_urls.register(r".*")
+def itp_map_check(url: str):
+    parent_class = get_parent_class(url)
+
+    if ("BaseMap" == parent_class):
+        if not ("itp_map://" in url):
+            warnings.warn("Warning, you are requesting a map file with this URL however, the protocol itp_map:// was not requested as part of the URL. The itp_map:// protocol is requiered for map corrections.")
+
+
+@check_urls.register(r".*posrec_models.*")
+def tf_check(url: str):
+    if not ("tf://" in url):
+        warnings.warn("Warning, you are requesting a position reconstruction model with this URL however, the protocol tf:// was not requested as part of the URL. The tf:// protocol is requiered for position reconstruction corrections.")
+
+
+@check_urls.register(r".*")
+def url_attr_check(url: str):
+    if not ("attr" in url):
```

Comment:
I havent run into any of those cases in the corrections so far, I think all the urls in global_v11 had this, and since its just a warning I think anyone digging deep enough to change the attribute should be fine?